PYME/recipes/measurement.py

```diff
@@ -389,9 +389,9 @@ def execute(self, namespace):
         pos0 = namespace[self.inputPositions]
         pos1 = namespace[self.inputPositions2 if self.inputPositions2 is not '' else self.inputPositions]
         if np.count_nonzero(pos0['z']) == 0 and np.count_nonzero(pos1['z']) == 0:
-            res = DistHist.distanceHistogram(pos0['x'], pos0['y'], pos1['x'], pos1['y'], self.nbins, self.binSize)
+            res = DistHist.distanceHistogramThreaded(pos0['x'], pos0['y'], pos1['x'], pos1['y'], self.nbins, self.binSize)
```

Comment:
Due to the fact that both the cluster and bakeshop will run `n_cores` recipes in parallel, recipe modules should be single threaded by default (although they may define an optional boolean parameter to run multithreaded).